automation/test.sh

```diff
@@ -16,7 +16,7 @@ trap teardown EXIT SIGINT SIGTERM SIGSTOP
 make cluster-down
 make cluster-up
 make cluster-sync
-test_args="-ginkgo.v -ginkgo.noColor -test.timeout 20m"
+test_args="-ginkgo.noColor"
```

Comment:
why you remove the 20m timeout from here and also the `ginkgo.v`?

Reply:
The timeout and verbose is now the default kept in Makefile. Originally, when you ran functional tests on your laptop it didn't show much.